rblxopencloud/http.py

```diff
@@ -118,7 +118,7 @@ def send_request(method: str, path: str, authorization: Optional[str]=None,
     else:
         body = response.text
 
-    if DEBUG:
+    if DEBUG == True:
```

Comment:
I've changed this to `VERSION_INFO == "alpha"` in uncommitted changes but for now it is okay.